presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -139,20 +145,26 @@
     private final AWSGlueAsync glueClient;
     private final Optional<String> defaultDir;
     private final String catalogId;
+    private final int numSegments;
+    private final Executor executor;
 
     @Inject
-    public GlueHiveMetastore(HdfsEnvironment hdfsEnvironment, GlueHiveMetastoreConfig glueConfig)
+    public GlueHiveMetastore(HdfsEnvironment hdfsEnvironment, GlueHiveMetastoreConfig glueConfig,
+            @ForGlueHiveMetastore Executor executor)
```

Comment:
nit: per code style, if there are too many params to put on one line, we put every on a separate line:
```
public GlueHiveMetastore(
    HdfsEnvironment hdfsEnvironment, 
    GlueHiveMetastoreConfig glueConfig,
    @ForGlueHiveMetastore Executor executor)
```

Reply:
Sure.

Reply:
This still needs updating